datalake/src/main/java/com/sequenceiq/datalake/flow/chain/DatalakeResizeFlowEventChainFactory.java

```diff
@@ -31,6 +34,11 @@ public String initEvent() {
     public FlowTriggerEventQueue createFlowTriggerEventQueue(DatalakeResizeFlowChainStartEvent event) {
         Queue<Selectable> chain = new ConcurrentLinkedQueue<>();
 
+        //take a backup
+        chain.add(new DatalakeTriggerBackupEvent(DATALAKE_TRIGGER_BACKUP_EVENT.event(),
+                event.getResourceId(), event.getUserId(), event.getBackupLocation(), "",
+                DatalakeBackupFailureReason.BACKUP_ON_RESIZE, event.accepted()));
```

Comment:
@kkalvagadda1 I noticed you also included event.accepted() when triggering the backup on an upgrade. But as far as I can tell that was a new Promise. Is there something I'm missing here?

Reply:
we can discuss about this in the standup.